gobase/intermediate-integer.go

```diff
@@ -20,7 +20,7 @@ func SaveArrUint8(path *string, data *[]uint8) bool{
 		return &final
 	}(data)
 
-	return SaveArr(path, *dataInString)
+	return saveArrCustom(path, *dataInString,1)
```

Comment:
please maintain the linting spaces from next time

Reply:
Yaa, sure I will keep in mind next time.